code/modules/mob/mob_movement.dm

```diff
@@ -12,7 +12,26 @@
 			return 1
 	return (!mover.density || !density || lying)
 
-
+//there are bugs with byond's default versions of these verbs
+/client/verb/moveup()
+	set name = ".moveup"
+	set instant = 1
+	Move(get_step(mob, NORTH), NORTH)
```

Comment:
If I did that, client/Move would never run, and thus mob delay would never happen, nor any of the other movement code in client/Move.